### DISCUSSION TITLE
How do you specify the attachments/storage path when running without Docker?

### DISCUSSION THREAD

**Discussion by danielrosehill** [sysadminsmedia/homebox#301] [Q&A]
I'm moving my Homebox installation over from a VM to an LXC and hoping to run it without docker.

My base structure is like this:

![Image](https://github.com/user-attachments/assets/0eac14fb-7b62-47cf-bbf3-ddda0875aacd)

It seems like `/documents` is the only folder that's populated and it's filled with all my attachments and images.

I tried the following launch commands

`homebox --web-port 3100 --storage-data /root/homebox/.data/attachments`

While it worked to pick up the SQLite DB, the attachments aren't being recognised/detected.

Any help much appreciated!

**Comment by tankerkiller125** [comment]
The storage of attachments is always going to be `<your specified path>/<guid>/documents/<guid>` and the SQLite file generally gets created at the root of `<your specified path>` along with it's various other files (the WAL file for example).

I need to get home on to my personal computer before I can check how the database itself stores the path, I believe it should be as `/documents/<blah>/<blah>...` but I'm not 100% sure, and I need to validate that it's not the full path.

**Reply by tankerkiller125** [reply]
OK, so after checking my dev environment, the database stores it the same way as the path I specified originally.

So what this means, is that if you originally set your path to say `homebox-data/` then the database has the entry as `homebox-data/<guid>/documents/<guid>.<type>` if you change the storage path at a later time, to say `container/` then the database will still be looking for `homebox-data/` and won't find the files.

You would need to update the database entries if you change the path at a later time.
